server/util.py

```diff
@@ -134,6 +136,56 @@ def construct_slack_attachment(
 
     return attachment
 
+def get_circle_amount(rdo):
+    """
+    Get the amount for a circle donation based on the three year total.
+    """
+    amount = rdo.amount
+    period = rdo.installment_period
+    amount = quantize(amount) / 3
+    if period == "monthly":
+        amount = amount / 12
+    return amount
+
+def send_cancellation_notification(contact, rdo, donation_type, reason, method):
+    """
+    Send a notification about a donation cancellation to Slack.
+    Rules:
+    - If the donation is a circle membership, notifications go to circle channel
+    - Only circle gets notfications for anything other than "cancellation_requested"
+    - For non-cirlce, if the reason is "cancellation_requested" we send to the general cancellations channel
```

Comment:
:way-to-picky: "non-circle" typo

Reply:
It's a good flag just to keep things polished! Thanks for spotting that. You saved me from git blame embarrassment 😄 (fixed in [75eec1d](https://github.com/texastribune/donations/pull/1176/commits/75eec1d7157619d24daea1016fa9af890614723b))